signal_analog/flow.py

```diff
@@ -940,3 +971,13 @@ class Ref(Arg):
 
     def __init__(self, arg):
         super(self.__class__, self).__init__(arg)
+
+
+class Dimensions(StreamMethod):
```

Comment:
Why do we have both the Dimensions class and the dimensions method?

Reply:
The class is the tree representing a dimensions method call and how to serialize it, the method call on the `Data` class is the DSL that an end user would use (and responsible for adding it to the call stack).